content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Newrelic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `Newrelic` trigger that scales based on a Newrelic metric.
+
+```yaml
+triggers:
+  - type: newrelic
```

Comment:
I'd go with this scaler name, based on the product name
```suggestion
  - type: new-relic
```

Reply:
That would match the overarching design for things like aws-cloudwatch etc where its Aws service Cloudwatch so a good callout. I've added this to my pr that I've done to the the original submitters request for adding the scaler.